### PR TITLE
so, this is basically a drop in the bucket. but fix at least SOME of …

### PR DESCRIPTION
**Pull request by meganschneider**
…the compile warnings, mostly ones around type declarations.

### CODE REVIEW

File: coordinatorlib-soql/src/main/scala/com/socrata/datacoordinator/common/soql/sqlreps/BooleanRep.scala [hunk 1]
```diff
@@ -27,7 +27,7 @@ class BooleanRep(val base: String) extends RepUtils with SqlPKableColumnRep[SoQL
       lit.asInstanceOf[SoQLBoolean].value.toString
     }.mkString(s"($base in (", ",", "))")
 
-  def count = "count(" + base + ")"
+  def count: String = "count(" + base + ")"
```
Comment (AEscobarCruz) [inline coordinatorlib-soql/src/main/scala/com/socrata/datacoordinator/common/soql/sqlreps/BooleanRep.scala]: s"count($base)" would be cleaner

File: coordinatorlib-soql/src/main/scala/com/socrata/datacoordinator/common/soql/sqlreps/DoubleRep.scala [hunk 2]
```diff
@@ -17,12 +17,12 @@ class DoubleRep(val base: String) extends RepUtils with SqlPKableColumnRep[SoQLT
...
   }
 
-  def dbl(v: SoQLValue) = v.asInstanceOf[SoQLDouble].value
+  def dbl(v: SoQLValue): Double = v.asInstanceOf[SoQLDouble].value
 
   def sql_in(literals: Iterable[SoQLValue]): String =
     literals.iterator.map(dbl).mkString(s"($base in (", ",", "))")
 
-  def count = "count(" + base + ")"
+  def count: String = "count(" + base + ")"
```
Comment (AEscobarCruz) [inline coordinatorlib-soql/src/main/scala/com/socrata/datacoordinator/common/soql/sqlreps/DoubleRep.scala]: Do these <SomeName>Rep extend a trait, they have this same method over and over.

File: dummy-secondary/src/main/scala/com/socrata/dummysecondary/DummySecondary.scala [hunk 3]
```diff
@@ -65,7 +69,7 @@ class DummySecondary(config: Config) extends Secondary[Any, Any] {
         ???
     }
     println("Current cookie: " + cookie)
-    readLine("New cookie? ") match {
+    readLine("New qcookie? ") match {
```
Comment (AEscobarCruz) [inline dummy-secondary/src/main/scala/com/socrata/dummysecondary/DummySecondary.scala]: change 'qcookie' back to 'cookie' here.

**Comment by AEscobarCruz** [comment]
other than those items above, lgtm

File: coordinatorlib-soql/src/main/scala/com/socrata/datacoordinator/common/soql/sqlreps/DoubleRep.scala (hunk 2 above)
Comment (meganschneider) [inline coordinatorlib-soql/src/main/scala/com/socrata/datacoordinator/common/soql/sqlreps/DoubleRep.scala]: Yes, but they (the reps in this dir) do not all explicitly define it. I don't know why.

**Comment by AEscobarCruz** [comment]
may need to be rebased, but looks good to me.

**Pull request merged**